frontend/src/app/applications/permit-application-list/permit-application-list.component.spec.ts

```diff
@@ -31,9 +33,10 @@ describe('PermitApplicationListComponent', () => {
           { provide: ApplicationService, useClass: ApplicationService },
           AlertService,
           { provide: XHRBackend, useClass: MockBackend },
-          { provide: Router, useValue: router }
+          { provide: Router, useValue: router },
+          { provide: AuthenticationService, useClass: MockService }
         ],
-        imports: [HttpModule, RouterTestingModule]
+        imports: [HttpModule, RouterTestingModule.withRoutes([{ path: 'admin/applications' }])]
```

Comment:
Does this need to handle both the admin the user case now?

Reply:
the withRoutes function wasn't actually doing anything, so I removed it.